stable/redis-ha/templates/redis-slave-statefulset.yaml

```diff
@@ -1,44 +0,0 @@
-apiVersion: apps/v1beta1
```

Comment:
Why did you delete this statefulset ?

Reply:
This functionality was merged into the master pod type to create the server pod type.

Reply:
Statefulsets stop auto-scaling in K8S when the first instance is orphaned. This causes problems when SS instances are re-labeled on the fly to move between load balancers. The compromise was to fall back to dynamic naming and use logic instead of hostname to determine which box should be the first master.